pkg/mcs/scheduling/server/cluster.go

```diff
@@ -252,7 +252,7 @@ func (c *Cluster) updateScheduler() {
 					zap.Strings("scheduler-args", scheduler.Args))
 				continue
 			}
-			if err := schedulersController.AddScheduler(s, scheduler.Args...); err != nil {
+			if err := schedulersController.AddScheduler(c.storage, s, scheduler.Args...); err != nil {
```

Comment:
The controller has a storage field internal, we don't need to pass it all around.

Reply:
Thx for ur information!